dev-docs/bidders/resetdigital.md

```diff
@@ -20,30 +20,37 @@ media_types: banner, video
 sidebarType: 1
 ---
 
-### bid params
+### Note
+
+Prebid adapter for Reset Digital requires approval and account setup. Video is supported but requires a publisher supplied renderer at this time.
```

Comment:
Any contact info or website for approval and setup?

Reply:
Sure, I will adjust it accordingly, thanks you.